src/Engine/Input/InputHandler.cpp

```diff
@@ -78,24 +82,166 @@ namespace gp1 {
+			{ { InputType::BUTTON, { InputLocation::KEYBOARD, buttons::keyKp5 } }, "Key.Kp5" },
+			{ { InputType::BUTTON, { InputLocation::KEYBOARD, buttons::keyKp6 } }, "Key.Kp6" },
+			{ { InputType::BUTTON, { InputLocation::KEYBOARD, buttons::keyKp7 } }, "Key.Kp7" },
+			{ { InputType::BUTTON, { InputLocation::KEYBOARD, buttons::keyKp8 } }, "Key.Kp8" },
+			{ { InputType::BUTTON, { InputLocation::KEYBOARD, buttons::keyKp9 } }, "Key.Kp9" },
+			{ { InputType::BUTTON, { InputLocation::KEYBOARD, buttons::keyKpDecimal } }, "Key.KpDecimal" },
+			{ { InputType::BUTTON, { InputLocation::KEYBOARD, buttons::keyKpDivide } }, "Key.KpDivide" },
+			{ { InputType::BUTTON, { InputLocation::KEYBOARD, buttons::keyKpMultiply } }, "Key.KpMultiply" },
+			{ { InputType::BUTTON, { InputLocation::KEYBOARD, buttons::keyKpSubtract } }, "Key.KpSubtract" },
+			{ { InputType::BUTTON, { InputLocation::KEYBOARD, buttons::keyKpAdd } }, "Key.KpAdd" },
+			{ { InputType::BUTTON, { InputLocation::KEYBOARD, buttons::keyKpEnter } }, "Key.KpEnter" },
+			{ { InputType::BUTTON, { InputLocation::KEYBOARD, buttons::keyKpEqual } }, "Key.KpEqual" },
+			{ { InputType::BUTTON, { InputLocation::KEYBOARD, buttons::keyLeftShift } }, "Key.LeftShift" },
+			{ { InputType::BUTTON, { InputLocation::KEYBOARD, buttons::keyLeftControl } }, "Key.LeftControl" },
+			{ { InputType::BUTTON, { InputLocation::KEYBOARD, buttons::keyLeftAlt } }, "Key.LeftAlt" },
+			{ { InputType::BUTTON, { InputLocation::KEYBOARD, buttons::keyLeftSuper } }, "Key.LeftSuper" },
+			{ { InputType::BUTTON, { InputLocation::KEYBOARD, buttons::keyRightShift } }, "Key.RightShift" },
+			{ { InputType::BUTTON, { InputLocation::KEYBOARD, buttons::keyRightControl } }, "Key.RightControl" },
+			{ { InputType::BUTTON, { InputLocation::KEYBOARD, buttons::keyRightAlt } }, "Key.RightAlt" },
+			{ { InputType::BUTTON, { InputLocation::KEYBOARD, buttons::keyRightSuper } }, "Key.RightSuper" },
+			{ { InputType::BUTTON, { InputLocation::KEYBOARD, buttons::keyMenu } }, "Key.Menu" },
+			{ { InputType::BUTTON, { InputLocation::MOUSE, buttons::mouseLeft } }, "Mouse.Left" },
+			{ { InputType::BUTTON, { InputLocation::MOUSE, buttons::mouseRight } }, "Mouse.Right" },
+			{ { InputType::BUTTON, { InputLocation::MOUSE, buttons::mouseMiddle } }, "Mouse.Middle" },
+			{ { InputType::BUTTON, { InputLocation::MOUSE, buttons::mouse1 } }, "Mouse.1" },
+			{ { InputType::BUTTON, { InputLocation::MOUSE, buttons::mouse2 } }, "Mouse.2" },
+			{ { InputType::BUTTON, { InputLocation::MOUSE, buttons::mouse3 } }, "Mouse.3" },
+			{ { InputType::BUTTON, { InputLocation::MOUSE, buttons::mouse4 } }, "Mouse.4" },
+			{ { InputType::BUTTON, { InputLocation::MOUSE, buttons::mouse5 } }, "Mouse.5" },
+			{ { InputType::BUTTON, { InputLocation::MOUSE, buttons::mouse6 } }, "Mouse.6" },
+			{ { InputType::BUTTON, { InputLocation::MOUSE, buttons::mouse7 } }, "Mouse.7" },
+			{ { InputType::BUTTON, { InputLocation::MOUSE, buttons::mouse8 } }, "Mouse.8" },
```

Comment:
Aren't there only 5 mouse buttons? left, right, middle, extra-1 and extra-2?

Reply:
According to GLFW there's 8 mouse buttons.
Even though most mouses only support 5 buttons.

Reply:
Ah, okay!